### PR TITLE
chore: update ignite config

### PR DESCRIPTION
**Pull request by ydcjeff**
# Pull request motivation(s)

PyTorch-Ignite now has own website: https://pytorch-ignite.ai which has how to guides and tutorials. We want Docsearch to work on that site.

### What is the current behaviour?

Search only applied to https://pytorch.org/ignite

*If the current behaviour is a bug, please provide all the steps to reproduce and screenshots with context.*

### What is the expected behaviour?

Search should show results from https://pytorch-ignite.ai

##### NB: Do you want to request a **feature** or report a **bug**?

Feature

##### NB2: Any other feedback / questions ?

Not sure the updated config is correct. If you could check, that would be great. Thanks!

### CODE REVIEW

**Comment by ydcjeff** [comment]
@shortcuts Can you please review this? Thanks!

File: configs/ignite.json [hunk 1]
```diff
@@ -9,22 +9,42 @@
           "js": "let versions=[]; for (let b of document.querySelectorAll('.rst-other-versions dd')){versions.push(b.innerText)};return JSON.stringify(versions)"
         }
       }
+    },
+    {
+      "url": "https://pytorch-ignite.ai/",
+      "selectors_key": "pytorch-ignite-ai"
     }
```
Comment (shortcuts) [inline configs/ignite.json:16]: You will see results from both websites on both search instances with this setup. If you'd like to limit the search to its website, you can use [`tags`](https://docsearch.algolia.com/docs/config-file/#using-custom-tags) for each URL group, and then in your frontend have:

on https://pytorch.org/ignite/
```jsx
<DocSearch
  // ... your credentials
  searchParameters={{
    facetFilters: ['tags:ignite']
  }}
/>
```

and on https://pytorch-ignite.ai/
```jsx
<DocSearch
  // ... your credentials
  searchParameters={{
    facetFilters: ['tags:ignite-ai']
  }}
/>
```

Comment (shortcuts) [reply configs/ignite.json:16]: concrete example

config: https://gist.github.com/shortcuts/b4b4e5ae8219a5e2496eb5fab2d2e1f6
live demo: https://codesandbox.io/s/bold-kate-g35nd?file=/src/App.js

**Comment by ydcjeff** [comment]
Thanks @shortcuts . I have updated the config.

**Pull request merged**